en/concepts/scene/light/shadow.md

```diff
@@ -55,7 +55,7 @@ ShadowMap renders the scene with the lights as the viewpoint. From the position
 | :--- | :--- |
 | **Enabled**         | Whether to enable the shadow effect. |
 | **Type**            | Choose the shadow type. |
-| **Saturation**      | Set the shadow saturation.  |
+| **Saturation**      | Set the shadow saturation, it is recommended to set as **1**. If need to reduce the saturation of the directional light shadows, we recommend to do it by increasing the ambient light instead of adjusting this value.  |
```

Comment:
```suggestion
| **Saturation**      | Set the shadow saturation. It is recommended to set as **1**. If it is necessary to reduce the saturation of the directional light shadows, it is recommended to do it by increasing the ambient light instead of adjusting this value.  |
```